packages/api-plugin-fulfillment-method-shipping-flat-rate/src/queries/flatRateFulfillmentMethod.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @name flatRateFulfillmentMethod
+ * @method
+ * @memberof Fulfillment/Queries
+ * @summary Query the Fulfillment collection for a single fulfillment method
+ * @param {Object} context - an object containing the per-request state
+ * @param {Object} input - Request input
+ * @param {String} input.methodId - The fulfillment method id
+ * @param {String} input.shopId - The shop id of the fulfillment method
+ * @returns {Promise<Object>} Mongo cursor
+ */
+export default async function flatRateFulfillmentMethod(context, input) {
+  const { collections } = context;
```

Comment:
These two destructing statements can be combined

Reply:
Done